src/main/java/no/nav/pto/veilarbportefolje/oppfolgingsbruker/OppfolgingsbrukerEntity.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.pto.veilarbportefolje.oppfolgingsbruker;
+
+import java.time.ZonedDateTime;
+
+public record OppfolgingsbrukerEntity(String aktoerid, String fodselsnr, String formidlingsgruppekode,
```

Comment:
as we mentioned on morning meeting, so many arguments can be bit tricky, since its easy to make mistake..maybe we can introduce builder, or divide arguments into subclasses...or leave it as it is and be extra careful when setting arguments.